docs/development.md

```diff
@@ -0,0 +1,222 @@
+# Development
```

Comment:
You can give an overview of the document at the top before listing the ToC as the reader should know what this document is about (this applies to other documents too). In that overview, you can also mention what `dev server` means.